src/content/popup/KanjiReferencesTable.fixture.tsx

```diff
@@ -0,0 +1,176 @@
+import { KanjiReferencesTable } from './KanjiReferencesTable';
+
+// import '../../../css/popup.css';
+
+const TEST_ENTRY = {
+  'c': '日',
+  'r': {
+    'py': [
+      'ri4'
+    ],
+    'on': [
+      'ニチ',
+      'ジツ'
+    ],
+    'kun': [
+      'ひ',
+      '-び',
+      '-か'
+    ],
+    'na': [
+      'あ',
+      'あき',
+      'いる',
+      'く',
+      'くさ',
+      'こう',
+      'す',
+      'たち',
+      'に',
+      'にっ',
+      'につ',
+      'へ'
+    ]
+  },
+  'm': [
+    'day',
+    'sun',
+    'Japan',
+    'counter for days'
+  ],
+  'rad': {
+    'x': 72,
+    'b': '⽇',
+    'k': '日',
+    'na': [
+      'ひ'
+    ],
+    'm': [
+      'day',
+      'sun',
+      'Japan',
+      'counter for days'
+    ],
+    'm_lang': 'en'
+  },
+  'refs': {
+    'nelson_c': 2097,
+    'nelson_n': 2410,
+    'halpern_njecd': 3027,
+    'halpern_kkld_2ed': 2606,
+    'heisig6': 12,
+    'henshall': 62,
+    'sh_kk2': 5,
+    'kanji_in_context': 16,
+    'busy_people': '1.A',
+    'kodansha_compact': 963,
+    'skip': '3-3-1',
+    'sh_desc': '4c0.1',
+    'conning': 1
+  },
+  'misc': {
+    'sc': 4,
+    'gr': 1,
+    'freq': 1,
+    'jlpt': 4,
+    'kk': 10,
+    'wk': 2,
+    'jlptn': 5
+  },
+  'm_lang': 'en',
+  'comp': [],
+  'cf': []
+}
+
+export default {
+  'even # rows last row filled': () => {
+    return (
+      <KanjiReferencesTable
```

Comment:
What might be nice here is to define a little wrapper component in this fixture file that simply sets the fixture background color based on the theme (e.g. has `<div style={{ background: 'var(--bg-color)' }}>`) and adds some padding so we can see the edges of the component (i.e. the rounding).

Alternatively, if you think it's more useful, we could actually just render all these four variants as the one fixture so you can easily compare them at a glance. If we did that, we probably wouldn't need the separate wrapper component.

Reply:
Yeah, I had something similar in mind as well. Will work this.